ddtrace/ext/__init__.py

```diff
@@ -1,5 +1,6 @@
-class AppTypes(object):
+PYTHON_STACK = "python.stack"
```

Comment:
this should go here: https://github.com/DataDog/dd-trace-py/blob/9faaf6dcd3ddaa6b46535f687ed06f2df82549bc/ddtrace/ext/errors.py#L10

But probably you can remove that line because you're not using that variable. We may add it later when you work on the `FIXME`.